src/physics/base/ParticleAccess.i.hh

```diff
@@ -39,41 +40,51 @@ CELER_INLINE_FUNCTION real_type ParticleAccess::speed() const
 //---------------------------------------------------------------------------//
 /*!
  * Lorentz factor [unitless].
+ *
  * Massless particles: Cannot be calculated (gamma = infinity)
  * Massive particles : E = gamma.mc^2 = K + mc^2 --> gamma = 1 + K/mc^2
  */
 CELER_INLINE_FUNCTION real_type ParticleAccess::lorentz_factor() const
 {
     REQUIRE(this->mass() > 0);
 
-    const real_type c_squared = constants::speed_of_light_squared;
-
-    real_type gamma;
-    real_type k_over_mc2;
-    k_over_mc2 = this->kinetic_energy() / this->mass() * c_squared;
-    gamma      = 1 + k_over_mc2;
-
-    return gamma;
+    real_type k_over_mc2 = this->kinetic_energy() / this->mass()
+                           * units::speed_of_light_sq;
```

Comment:
Looks like we need a parenthesis here:
`this->kinetic_energy() / (this->mass() * units::speed_of_light_sq)`

Reply:
Good catch -- I was blindly copy-pasting the original `k_over_mc2 = this->kinetic_energy() / this->mass() * c_squared;` :)